packages/phishing-controller/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.1]
+### Uncategorized
+- Release 81.0.0 ([#1747](https://github.com/MetaMask/core/pull/1747))
+- devDeps: @metamask/eslint-config*->12.1.0 ([#1740](https://github.com/MetaMask/core/pull/1740))
```

Comment:
- We should give a name to this section
- This is showing "Release 81.0.0" because the previous made some dependency bumps to this package which weren't released (intentional) — but we need to be more clear about what those changes are
- We can remove the `devDeps` entry since it's development-only

Putting this together:

``` suggestion
### Changed
- Bump dependency on `@metamask/base-controller` to ^3.2.3 ([#1747](https://github.com/MetaMask/core/pull/1747))
- Bump dependency on `@metamask/controller-utils` to ^5.0.2 ([#1747](https://github.com/MetaMask/core/pull/1747))
```